lib/web_ui/lib/src/engine/html/bitmap_canvas.dart

```diff
@@ -936,7 +936,16 @@ class BitmapCanvas extends EngineCanvas {
   void drawParagraph(EngineParagraph paragraph, ui.Offset offset) {
     assert(paragraph.isLaidOut);
 
-    if (paragraph.drawOnCanvas && _childOverdraw == false) {
+    /// - paragraph.drawOnCanvas checks that the text styling doesn't include
+    /// features that prevent text from being rendered correctly using canvas.
+    /// - _childOverdraw check prevents sandwitching multiple canvas elements
+    /// when we have alternating paragraphs and other drawing commands that are
+    /// suitable for canvas.
+    /// - To make sure an svg filter is applied correctly to paragraph we
+    /// check isInsideSvgFilterTree to make sure dom node doesn't have any
+    /// parents that apply one.
+    if (paragraph.drawOnCanvas && _childOverdraw == false &&
+        _renderStrategy.isInsideSvgFilterTree == false) {
```

Comment:
nit: `!_renderStrategy.isInsideSvgFilterTree` instead of `== false`. `!_childOverdraw` too.

Reply:
Done